src/Context/processOptions.ts

```diff
@@ -0,0 +1,42 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import { Platform } from 'react-native'
+import type { CacheOptions, Options } from './types'
+
+export function processOptions(options: Options): Options {
+  return {
+    ...processStreamingOptions(options),
+    cache: processCacheOptions(options?.cache)
+  }
+}
+
+function processStreamingOptions(options: Options): Options {
+  if (
+    (typeof options?.streamEnabled === 'undefined' || options.streamEnabled) &&
+    Platform.OS === 'android'
+  ) {
+    return {
+      ...options,
+      streamEnabled: false,
+      pollingEnabled: options?.pollingEnabled !== false
```

Comment:
is this part disabling some streaming/polling functionality if it's Android? Is that just a limitation of that OS?

Reply:
There's a strange issue with React Native running on Android. Streaming with `XMLHttpRequest` just never resolves. I tried using `fetch` instead, but Meta never implements the streaming part of `fetch` in React Native. So with RN on Android, we switch from streaming to polling.